waspc/cli/src/Wasp/Cli/Command/Telemetry/Common.hs

```diff
@@ -25,4 +22,7 @@ ensureTelemetryCacheDirExists = do
   return telemetryCacheDirPath
 
 getTelemetryCacheDirPath :: Path' Abs (Dir UserCacheDir) -> Path' Abs (Dir TelemetryCacheDir)
-getTelemetryCacheDirPath userCacheDirPath = userCacheDirPath SP.</> [reldir|wasp/telemetry|]
+getTelemetryCacheDirPath userCacheDirPath = getWaspCacheDirPath userCacheDirPath SP.</> [reldir|telemetry|]
+
+getWaspCacheDirPath :: Path' Abs (Dir UserCacheDir) -> Path' Abs (Dir TelemetryCacheDir)
+getWaspCacheDirPath userCacheDirPath = userCacheDirPath SP.</> [reldir|wasp|]
```

Comment:
Don't forget about Strong Path conventions: https://github.com/wasp-lang/wasp/pull/953#discussion_r1071593511.

TL;DR: Drop the `SP.` prefix.